kimchi/src/tests/and.rs

```diff
@@ -143,6 +143,36 @@ where
         .unwrap();
 }
 
+/// Generic test for checking serialization & regression of AND circuit.
+fn prove_and_check_serialization_regression<G: KimchiCurve, EFqSponge, EFrSponge>(
+    bytes: usize,
+    buf_expected: Vec<u8>,
+) where
+    G::BaseField: PrimeField,
+    EFqSponge: Clone + FqSponge<G::BaseField, G, G::ScalarField>,
+    EFrSponge: FrSponge<G::ScalarField>,
+{
+    let rng = &mut StdRng::from_seed(RNG_SEED);
```

Comment:
For a follow up: use `o1_utils::tests:make_test_rng`. More random tests this way.

Reply:
Opened https://github.com/o1-labs/proof-systems/issues/2610

Reply:
@dannywillems the whole point of this test is to use a fixed RNG because we want this to be static and reproducible. I'm not sure what you mean?

Reply:
Right. Sorry, it is a regression test. Not a valid comment.